federation/Makefile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DBG_MAKEFILE ?=
+ifeq ($(DBG_MAKEFILE),1)
```

Comment:
Not entirely sure. Assuming FreeBSD actually ships BSD make, this should work - http://www.freebsd.org/cgi/man.cgi?make(1). I don't have access to a Mac, so I need your help here.

Also, this isn't specific to federation, the top-level Makefile (the one in the root directory of the source) also has this, so I think this should work on Mac.